Resources/public/js/main.js

```diff
@@ -0,0 +1,28 @@
+require.config({
+    paths: {
+        examplenews: '../../examplenews/js',
+        examplenewscss: '../../examplenews/css'
+    }
+});
+
+define(function() {
+
+    'use strict';
+
+    return {
+
+        name: "Example News Bundle",
+
+        initialize: function(app) {
+
+            app.components.addSource('examplenews', '/bundles/examplenews/js/components');
+
+            app.sandbox.mvc.routes.push({
+                route: 'example/news',
```

Comment:
defined in the first tutorial of this series - https://github.com/sulu-io/ExampleNewsBundle/pull/1/files#diff-656b9372aac1b41132a33f5460c065a8R21